### ISSUE TITLE
Add support for Forestry and static content

### ISSUE THREAD

**Issue by seancdavis** [ample/gatsby-starter-ample#41]
Knowing Forestry is our default for the time being, let's put our static content approach together here. We can start with some basic models that we use on most projects.

- [x] Flexible page model. Perhaps we have a default page (template), container (section), and content or button (component) so we have something that works out of the box.
- [x] SEO
- [ ] Header and footer menus
- [x] Redirects
- [ ] Forms 
- [ ] Fixtures (i.e. content for dev)

**Comment by seancdavis** [comment]
Thinking out loud:

- [ ] Flexible page model. Perhaps we have a default page (template), container (section), and content or button (component) so we have something that works out of the box.
- [ ] SEO
- [ ] Header and footer menus

**Comment by seancdavis** [comment]
After digging into this, I've decided to skip menus, forms, and fixtures for now, in lieu of the following actions:

- For menus, I need to have a couple conversations before making too many inferences in this starter.
- For forms, it's a big task and we haven't actually implemented it. Therefore, I'm going to add an issue to track it, but we won't work it into the starter until we build one that it production ready.
- I want to think about how we're using fixtures. We have an opportunity to work with real data right out of the gate, but that may not be suitable for working with components in Storybook. I want to think this through, but don't want to hold up the progress I've made to date. I likely haven't done anything we'll have to undo.
